content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -92,26 +94,27 @@ Although, Poseidon-Firmament scheduler is capable of scheduling various types of
 |Support for Ephemeral Storage, in addition to CPU/Memory|Y|Y||
 
 
-## Installation	
-
-For in-cluster installation of Poseidon, please start at the [Installation instructions](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/install/README.md).
-
+## Installation
 
-## Development	
+[Installation](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/install/README.md#Installation) explains how to deploy Poseidon/Firmament to your cluster.
 
-For developers, please refer to the [Developer Setup instructions](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/devel/README.md).
```

Comment:
@sftim Any particular reason for removing this?

Reply:
I've added:
```markdown
* See [Poseidon/Firmament](https://github.com/kubernetes-sigs/poseidon#readme) on GitHub for more information.
```
which links to the developer information.